templates/component.html

```diff
@@ -6,10 +6,13 @@
         <div class="hxRow">
             <div class="hxCol-8">
                 <div class="hxRow">
-                    <div class="hxCol">
+                    <div class="hxCol-9">
```

Comment:
Avoid using the hx grid for all layout needs. 

hxRows add extra bottom margin that throws off desired layouts when nested, and it makes the markup more complex.   Instead, try using more traditional css tactics, like floating elements to get the desired layout.

**Also,** there is a bug in the hxGrid that causes buttons to stretch to fill 100% of the horizontal space of the parent hxCol.  To overcome this we have to break the flex inheritance.

**Here's a screenshot highlighting these issues:**

![image](https://user-images.githubusercontent.com/933210/33394756-f70d3c8e-d508-11e7-9406-48f35948ecc1.png)

**To fix these issues, consider this markup:**

![image](https://user-images.githubusercontent.com/933210/33395054-f4aa3658-d509-11e7-95d8-716f4934e79e.png)

This removes the nested hxRow and columns, and adds a dummy div around the title, button and blurb.  The dummy div breaks the flex inheritance that causes the button to stretch.

Consider adding a .pull-right class to the _utilities.scss.

**Here's what it looks like after the change:**

![image](https://user-images.githubusercontent.com/933210/33395121-223c26ee-d50a-11e7-9a20-9dcb4bf4fa8d.png)